src/conn/conn_prefetch.c

```diff
@@ -105,6 +105,9 @@ __wt_prefetch_thread_run(WT_SESSION_IMPL *session, WT_THREAD *thread)
         if (pe == NULL)
             break;
 
+        /* Add this dhandle to the list of dhandles prefetch threads are currently working on. */
+        conn->prefetch_dhandles[thread->id] = pe->dhandle;
```

Comment:
I think this should be a `WT_PUBLISH`.

Reply:
Done [ae383e8](https://github.com/wiredtiger/wiredtiger/pull/10189/commits/ae383e82604d374efa961601fb9e9406cf492771)